dockstore-integration-testing/src/test/java/io/dockstore/client/cli/MockedIT.java

```diff
@@ -81,7 +80,6 @@
 
     @Before
     public void clearDB() throws Exception {
-        CommonTestUtilities.getTestingPostgres().clearDatabase();
```

Comment:
I may have missed it, is 
a) the database being cleared elsewhere
b) is your intent here that the database doesn't need to be cleared for these particular tests?

Reply:
There is no database here, everything is mocked.